src/components/oppgaver/DokumentasjonKravView.tsx

```diff
@@ -103,19 +108,34 @@ const DokumentasjonKravView: React.FC<Props> = ({dokumentasjonkrav, dokumentasjo
 
     const onSendClicked = (event: React.SyntheticEvent) => {
         event.preventDefault();
-        if (!fiksDigisosId || overMaksStorrelse) {
+        if (!fiksDigisosId || !dokumentasjonkrav) {
             return;
         }
         setIsUploading(true);
         setErrorMessage(undefined);
+        setOverMaksStorrelse(false);
+        setFileUploadingBackendFailed(false);
         const path = innsynsdataUrl(fiksDigisosId, InnsynsdataSti.VEDLEGG);
+        let formData: any = undefined;
 
         if (Object.keys(dokumentasjonkravFiler).length === 0) {
             setErrorMessage("vedlegg.minst_ett_vedlegg");
             fileUploadFailedEvent("vedlegg.minst_ett_vedlegg");
             setIsUploading(false);
         }
 
+        const handleFileUploadFailedInBackend = (fileBackendResponse: Fil[], reference: string) => {
```

Comment:
reference er et litt vagt variabelnavn - hva slags referanse til hva?

Reply:
Reference i dette tilfellet blir bare brukt som en forkortelse for dokumentasjonkravReferanse, men ja det tenkte jeg ikke over. Det kan bli ordnet med neste refaktorering.